assets/email_template.html

```diff
@@ -4,7 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <meta name="description" content="nf-core/marsseq: MARS-seq 1/2 preprocessing pipeline">
+  <meta name="description" content="nf-core/marsseq: MARS-seq v2 preprocessing pipeline">
```

Comment:
isn't it v1?

Reply:
It's v2 only now, there is also v1, but I dropped the support for it as it is not used and requires custom alignment. It also has different cell barcoding scheme which is worse than v2. This way it's less confusing.